### ISSUE TITLE
[Accessibility] Default contrast is too low

### ISSUE THREAD

**Issue by steventrouble** [emilk/egui#2005]
**Describe the bug**
The BG/FG contrast in the default light theme are too low to meet AAA accessibility standards:

https://webaim.org/resources/contrastchecker/?fcolor=585858&bcolor=E6E6E6

Button:
![Example button](https://user-images.githubusercontent.com/605676/188510290-2c0aa8a1-4aad-45f9-9209-c6130cf6d81e.png)

The issue seems to be that, even thought the text color is set to #000, it's rendering as #585858.

**To Reproduce**
1. Create a button using the default light theme.

**Expected behavior**
The text color should be black to be accessible to people with vision problems.

**Desktop:**
 - OS: Windows 11

**Comment by goldwind-ting** [comment]
```rust
#![cfg_attr(not(debug_assertions), windows_subsystem = "windows")] // hide console window on Windows in release
use eframe::egui;
use egui::Color32;
use egui::FontFamily::Monospace;
use egui::FontId;
use egui::TextStyle::*;

fn main() {
    let options = eframe::NativeOptions::default();
    eframe::run_native(
        "My egui App",
        options,
        Box::new(|_cc| Box::new(MyApp::default())),
    );
}

struct MyApp;

impl Default for MyApp {
    fn default() -> Self {
        Self
    }
}

impl eframe::App for MyApp {
    fn update(&mut self, ctx: &egui::Context, _frame: &mut eframe::Frame) {
        let mut visuals = egui::Visuals::light();

        visuals.widgets.inactive.bg_fill = Color32::WHITE;
        visuals.widgets.inactive.bg_stroke.color = Color32::RED;
        visuals.widgets.inactive.fg_stroke.color = Color32::BLACK;
        visuals.widgets.inactive.rounding = 1.0.into();

        visuals.widgets.hovered.bg_fill = Color32::WHITE;
        visuals.widgets.hovered.bg_stroke.color = Color32::RED;
        visuals.widgets.hovered.fg_stroke.color = Color32::BLACK;
        visuals.widgets.hovered.rounding = 1.0.into();

        let mut style = (*ctx.style()).clone();
        style.text_styles = [(Button, FontId::new(25.0, Monospace))].into();
        ctx.set_style(style);
        ctx.set_visuals(visuals);
        egui::CentralPanel::default().show(ctx, |ui| ui.button("click me!"));
    }
}

```
@steventrouble  Hi, please try this.

**Comment by steventrouble** [comment]
All of those together fix it, but when I lower the font size the text becomes grey again.

All of the above:
![image](https://user-images.githubusercontent.com/605676/194478029-31bd7ea8-2c69-4a59-a3b5-36edb3ab7011.png)

With the font back to normal:
![image](https://user-images.githubusercontent.com/605676/194478004-716877a1-8538-43cf-a9a4-2a71ec7e7932.png)

Is this an issue with font aliasing? Any way I can turn that off or tweak the antialiasing method?

**Comment by emilk** [comment]
@steventrouble please try latest `master` - I recently improved text rendering in https://github.com/emilk/egui/pull/2071
